test/syntax.jl

```diff
@@ -925,8 +925,8 @@ g21054(>:) = >:2
 @test g21054(-) == -2
 
 # issue #21168
-@test Meta.lower(Main, :(a.[1])) == Expr(:error, "invalid syntax \"a.[1]\"")
-@test Meta.lower(Main, :(a.{1})) == Expr(:error, "invalid syntax \"a.{1}\"")
+@test_broken Meta.lower(Main, :(a.[1])) == Expr(:error, "invalid syntax \"a.[1]\"")
+@test_broken Meta.lower(Main, :(a.{1})) == Expr(:error, "invalid syntax \"a.{1}\"")
```

Comment:
Seems a regression

Reply:
Maybe good with an issue at JuliaSyntax.jl so there is something tracking this.

Reply:
https://github.com/JuliaLang/JuliaSyntax.jl/issues/532